### DISCUSSION TITLE
How to limit the nested levels shown in the toc?

### DISCUSSION THREAD

**Discussion by dgnemo** [imfing/hextra#428] [Q&A]
Does hextra support limiting the nested levels shown in the toc?

Thanks!

**Comment by imfing** [comment]
It's not supported yet. I think maybe we should respect Hugo's `tableOfContents` settings https://gohugo.io/methods/page/tableofcontents/ 🤔

**Reply by dgnemo** [reply]
If respecting Hugo `tableOfContents` setting is too hard I think it's fair to have a theme setting ...

Actually I think it would be even more useful if the user could customize it on a per-page basis 🤔
